frontend/styles/components/docs/DocsShell.module.scss

```
@@ -0,0 +1,407 @@
+  min-width: 0;
+  height: 100%;
+  position: relative;
+}
+
+.article {
+  width: 100%;
+  height: 100%; // own independent scroll pane -- see .columns's comment
+  overflow-y: auto;
+  padding-top: 4px;
+
+  :global(h1) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)